src/lib/env.js

```diff
@@ -85,11 +87,11 @@ export class Env {
     };
 
     const langToFetch = lang.src === 'pseudo' ?
-      { code: this.defaultLang, src: 'app' } :
+      { code: 'en-US', src: 'app' } :
```

Comment:
ugh, do you really need the lang code to be hardcoded that deep in the codebase? I'd prefer to set the source language during Env initialization.